test/test.auth.ts

```diff
@@ -15,8 +15,9 @@ import * as assert from 'power-assert';
 import * as async from 'async';
 import * as nock from 'nock';
 import utils from './utils';
-let googleapis = require('../');
-let googleauth = require('../lib/googleauth.js');
+import googleauth from '../lib/googleauth';
+const googleapis = require('../');
```

Comment:
Right now googleapis.ts doesn't currently export an es6 module.  That change is very involved, and best kept separate from this one.